lnurl.py

```diff
@@ -64,13 +63,42 @@ def xor_decrypt(key, blob):
     name="lnurldevice.lnurl_v1_params",
 )
 async def lnurl_v1_params(
+    request: Request,
+    device_id: str = Query(None),
+    p: str = Query(None),
+    atm: str = Query(None),
+    gpio: str = Query(None),
+    profit: str = Query(None),
+    amount: str = Query(None),
+):
+    return await lnurl_params(request, device_id, p, atm, gpio, profit, amount)
+
+
+@lnurldevice_ext.get(
+    "/api/v2/lnurl/{device_id}",
+    status_code=HTTPStatus.OK,
+    name="lnurldevice.lnurl_v2_params",
+)
+async def lnurl_v2_params(
     request: Request,
     device_id: str = Query(None),
     p: str = Query(None),
     atm: str = Query(None),
     pin: str = Query(None),
     amount: str = Query(None),
     duration: str = Query(None),
```

Comment:
here, changed to pin - amount - duration instead auf gpio - profit - amount

Reply:
we use amount all the time for invoices, i think using amount for the duration how long the switch is triggered is misleading.

Reply:
The option to change the time and amount afterwards really useful. This is very welcome. It should be compatible with the already install bitcoinSwitch. There are quite a few already. In so far this is a good solution. 

The display of the extension version is really useful. The warning would be perhaps additionally also with the selection and option to install still appropriate. Then you know immediately what you have to adjust and can warn the user if necessary.